operators/pkg/tenant-controller/tenant_controller.go

```diff
@@ -177,19 +247,25 @@ func (r *TenantReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		tn.Status.Subscriptions["keycloak"] = crownlabsv1alpha2.SubscrOk
 	}
 
-	if nsOk {
-		if err = r.handleNextcloudSubscription(ctx, &tn, nsName); err != nil {
-			klog.Errorf("Error when updating nextcloud subscription for tenant %s -> %s", tn.Name, err)
-			tn.Status.Subscriptions["nextcloud"] = crownlabsv1alpha2.SubscrFailed
-			retrigErr = err
-			tnOpinternalErrors.WithLabelValues("tenant", "nextcloud").Inc()
+
+	if keepNsOpen { // Only handle nextcloud subscription if namespace exists
+	    if nsOk {
```

Comment:
Should be merged in one if statement.

Reply:
I would disagree, because `keepNsOpen==false` should cause `klog.Infof("Nextcloud subscription for tenant %s not updated because namespace has been temporarily deleted", tn.Name)` while if `keepNsOpen==true and nsOk==false` should `cause klog.Errorf("Could not handle nextcloud subscription for tenant %s -> namespace update for secret gave error", tn.Name)`

Reply:
@giorio94 I will wait for your reply here before moving ahead

Reply:
I would honestly just skip printing the log message namespace creation is disabled, but if you prefer to keep it as now that is also fine on my side. BTW, most likely, nextcloud will be removed in a near future.

Reply:
Ok I will leave it, sounds good :)